packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -119,8 +119,8 @@ export interface LibraryInfo {
 /**
  * A pointer from the current span to another span in the same trace or in a
  * different trace. For example, this can be used in batching operations, where
- * a single batch handler processes multiple requests from different traces or
- * when the handler receives a request from a different project.
+ * a single batch instrument processes multiple requests from different traces or
+ * when the instrument receives a request from a different project.
```

Comment:
This is not right, looks like this got replaced due to general find and replace stuff. Please revert back the original comment here.